src/hint_processor/uint_utils.zig

```diff
@@ -0,0 +1,23 @@
+const std = @import("std");
+const Felt252 = @import("../math/fields/starknet.zig").Felt252;
+
+pub fn split(num: u512, comptime N: usize, num_bits_shift: u32) [N]Felt252 {
```

Comment:
Here we should deal with big integers, meaning some integers composed of limbs of u64. As we don't have such a library at the moment (but soon), can you use the dedicated std big int library `std.math.big.int`? Here you will find tools to play with big integers via limbs.